src/utils/base64.ts

```diff
@@ -0,0 +1,10 @@
+import { Base64 } from "js-base64";
+import { HoloHash, HoloHashB64 } from "../types.js";
+
+export function deserializeHash(hash: HoloHashB64): HoloHash {
```

Comment:
Can we rename these to `encodeHashTo64` and `decodeHashFromB64` or something similar? It has nothing to do with serialization.